src/baseline.rs

```diff
@@ -200,6 +234,8 @@ fn generate_rustdoc(
     match crate_source {
         CrateSource::Registry { .. } => {
             // Clean up after ourselves.
+            let cache_dir = cache_dir.unwrap();
+            let cached_rustdoc = cached_rustdoc.unwrap();
```

Comment:
Please use `.expect()` with a suitable message on both. It isn't obvious why we should expect these to be `Some`, and it will only be less obvious over time as this code gets more complex, more refactored, and less fresh in our minds.

Reply:
Done in 80e56feaec2acb6aeaabac74375dbfb8733ffe7a.